docker.sh

```diff
@@ -1,19 +1,24 @@
 #!/usr/bin/env bash
 
-TASK_NAME=$1;
+PROJECT_TASK_NAME=$1;
+PROJECT_PHP_VERSION=${2:-71};
```

Comment:
why `TASK_NAME` has been divided to `PROJECT_TASK_NAME` and `PROJECT_PHP_VERSION`?